component/src/test/java/org/wso2/extension/siddhi/io/http/source/HttpsSourceTestCaseForSSL.java

```diff
@@ -46,217 +46,181 @@
 
-        executionPlanRuntime.addCallback("query1", new QueryCallback() {
-            @Override
-            public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
-                EventPrinter.print(timeStamp, inEvents, removeEvents);
-                for (Event event : inEvents) {
-                    receivedEventNameList.add(event.getData(0).toString());
-                }
-            }
-        });
-        executionPlanRuntime.start();
-        // publishing events
-        List<String> expected = new ArrayList<>(2);
-        expected.add("John");
-        expected.add("Mike");
-        String event1 =
-                "<events><event><name>John</name>" + "<age>100</age><country>Sri Lanka</country></event></events>";
-        String event2 = "<events><event><name>Mike</name>" + "<age>20</age><country>USA</country></event></events>";
-        new HttpTestUtil().httpsPublishEvent(event1, "https://localhost:9095/endpoints/RecPro", false,
-                "text/plain");
-        new HttpTestUtil().httpsPublishEvent(event2, "https://localhost:9095/endpoints/RecPro", false,
-                "text/plain");
-        Thread.sleep(100);
-        Assert.assertEquals(receivedEventNameList.toString(), expected.toString());
-        executionPlanRuntime.shutdown();
-    }
-    /**
-     * Creating test for publishing events with https protocol with invalid keystore.
-     * @throws Exception Interrupted exception
-     */
-    @Test
-    public void testHTTPSInputTransportInvalidKeyStore() throws Exception {
-        logger.info("Creating test for publishing events with https protocol with invalid keystore.");
-        new HttpTestUtil().setCarbonHome();
-        masterConfigs.clear();
-        masterConfigs.put("source.http.https.keystore.file", "${carbon.home}/conf/security/store.jks");
-        masterConfigs.put("source.http.https.keystore.pass", "wso2carbon");
-        masterConfigs.put("source.http.https.cert.pass", "wso2carbon");
-
-        receivedEventNameList = new ArrayList<>(2);
-        SiddhiManager siddhiManager = new SiddhiManager();
-        InMemoryConfigManager inMemoryConfigManager = new InMemoryConfigManager(masterConfigs);
-        inMemoryConfigManager.generateConfigReader("source", "http");
-        siddhiManager.setConfigManager(inMemoryConfigManager);
-        siddhiManager.setExtension("xml-input-mapper", XmlSourceMapper.class);
-        String inStreamDefinition = "" + "@source(type='http', @map(type='xml'), "
-                + "receiver.url='https://localhost:9095/endpoints/RecPro', " + "basic.auth.enabled='false'" + ")"
-                + "define stream inputStream (name string, age int, country string);";
-        String query = ("@info(name = 'query1') " + "from inputStream " + "select *  " + "insert into outputStream;");
-        ExecutionPlanRuntime executionPlanRuntime = siddhiManager
-                .createExecutionPlanRuntime(inStreamDefinition + query);
+//    /**
```

Comment:
Let's remove commented out code

Reply:
Fixed in 602e96bfa91ea786afe9f9bd414b1bc8b1787631